### DISCUSSION TITLE
Welcome to tardis Discussions!

### DISCUSSION THREAD

**Discussion by wkerzendorf** [tardis-sn/tardis#1373] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Krishna-web-hub** [comment]
@wkerzendorf  hello sir i am new to this tardis collaboration and i am finding issues in pushing the change in issue #3290

**Comment by mansi-tomar-sudo** [comment]
Hi! I'm Mansi, a 2nd-year AI/ML student interested in contributing to TARDIS for GSoC 2027.

I've started going through the TARDIS codebase and developer documentation, and I'm currently looking for a small contribution to get familiar with the project and workflow.

I noticed a couple of small documentation issues and have already asked about fixing them on PR #3615. I'm also exploring the developer workflow/docs to understand the codebase better.

I'd appreciate any guidance on where a beginner contributor could start contributing meaningfully to TARDIS.
